tests/test_macos.py

```diff
@@ -17,7 +32,17 @@
         pytest.param({"appname": "foo", "version": "v1.0"}, id="app_name_version"),
     ],
 )
-def test_macos(params: dict[str, Any], func: str) -> None:
+def test_macos(mocker: MockerFixture, params: dict[str, Any], func: str) -> None:
+    # Make sure we are not in Homebrew
+    py_version = sys.version_info
+    mocker.patch(
+        "sys.prefix",
+        (
+            "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions"
+            f"/{py_version.major}.{py_version.minor}"
```

Comment:
save these two lines into a variable, and call mocker.patch with it to make this more compact